packages/cli/tests/subjects/custom-webpack/template.html

```diff
@@ -2,13 +2,11 @@
 <html lang="en">
 	<head>
 		<meta charset="utf-8">
-		<title><%= htmlWebpackPlugin.options.title %></title>
+		<title><%= cli.title %></title>
```

Comment:
Guess this is a bit of a breaking change, do you think we can just publish it as a minor?

Reply:
Ah shoot, good catch. Definitely can't get away with that.

I don't think I touched any of this, so not really familiar with how the plugin actually works, but I'll see if I can wrangle this into a non-breaking change. No idea if that's possible, but we'll see.

Reply:
Okay, was able to fix this. It's a tad messy in the source, but should be non-breaking. Tested by adding adding a `<%= JSON.stringify(htmlWebpackPlugin.options) %>` and comparing, more or less.

Reply:
Updated the `custom-template` test to ensure access via `htmlWebpackPlugin.options...` still works as intended.

https://github.com/preactjs/preact-cli/blob/66ab462654c56dcee3ee89a24b7f3e8d2351c96f/packages/cli/tests/subjects/custom-template/template.html#L6-L8

However, **there are still a couple differences on the `htmlWebpackPlugin` object that are breaking**, but I'm tempted to ignore these as they require a very specific use case in order to surface and should be trivial to fix. [preact-www runs into one of these](https://github.com/preactjs/preact-www/blob/5e23416a196929a81ef606beb0c59b7c989d8284/src/template.html#L46), as `htmlWebpackPlugin.files.chunks` no longer exists.

Here's a cleaned up version of the diff between master (left) and this PR (right): https://www.diffchecker.com/E1JDpVpK